src/moneyTracker.js

```diff
@@ -0,0 +1,63 @@
+function Moneytracker(date, credit, debit, balance) {
+  this.date = "";
+  this.credit = 0;
+  this.debit = 0;
+  this.balance = 0;
+  this.MIN_BALANCE = 0;
+  // this.displayMessage = "Welcome to Moneytracker";
+};
+
+var transactions = [];
+
+Moneytracker.prototype.deposit = function (money, dateOfTransaction) {
+  this.credit += money;
+  this.insertDate(dateOfTransaction);
+  this.balance += money;
+  this._createTransaction();
+};
+
+Moneytracker.prototype.withdraw = function (money, dateOfTransaction) {
+  // this.checkBalance();
+  this.debit -= money;
+  this.insertDate(dateOfTransaction);
+  this.balance -= money;
+  this._createTransaction();
+};
+
+Moneytracker.prototype.insertDate = function (dateOfTransaction) {
+  this.date = dateOfTransaction;
+};
+
+Moneytracker.prototype._createTransaction = function () {
+  var transaction = {
+      // date: new Date().toLocaleDateString(),
+      date: this.date,
+      credit: String(this.credit),
```

Comment:
Do we want `credit` and `debit` and `balance` stored as strings? Are they strings or numbers?